apps/emqx/src/emqx_schema.erl

```diff
@@ -3355,6 +3355,8 @@ default_listener(SSLListener) ->
 %% otherwise always return string.
 naive_env_interpolation(undefined) ->
     undefined;
+naive_env_interpolation(<<>>) ->
+    <<>>;
```

Comment:
undefined ?

Reply:
I think use empty string to indicate there is no bootstrap file is reasonable, and we also need to handle the empty string clause